### DISCUSSION TITLE
More Responsible AI resources

### DISCUSSION THREAD

**Discussion by MatthewCalder-msft** [msusdev/Innovation-Challenge#16] [General]
Good set of resources to help you with the concepts and tools for developing AI powered applications responsibly
[Responsible AI Resources for Developers](https://techcommunity.microsoft.com/t5/microsoft-developer-community/responsible-ai-resources-for-developers/ba-p/4189381)
[New Responsible AI Features for Building Trustworthy AI](https://techcommunity.microsoft.com/t5/microsoft-developer-community/new-responsible-ai-features-for-building-trustworthy-ai/ba-p/4257321)
[Evaluating generative AI: Best practices for developers](https://techcommunity.microsoft.com/t5/microsoft-developer-community/evaluating-generative-ai-best-practices-for-developers/ba-p/4271488)
[MS Learn official collection Building and using AI models responsibly](https://learn.microsoft.com/en-us/collections/wrdxbjpj34m1kz)

**Comment by nermeennasim** [comment]
Thankyou Mathew
